### PR TITLE
Test nanopore merge

### PR DESCRIPTION
**Pull request by lculibrk**
Adding ONT support and minor changes to enable hg38 bams with "chr" in chromosome names to run through the pipeline.

### CODE REVIEW

**Review by dustinbleile — APPROVED** [review]

File: Snakefile [hunk 1]
```diff
@@ -56,6 +58,17 @@ def devtools_install():
         devtools_cmd = f"devtools::install_github('lculibrk/Ploidetect', ref = '{ver}')"
     return f'"{devtools_cmd}"'
 
+rule download_cytobands:
+    """Downloads cytoband data for plotting & (todo) hgver-specific centromere filtering"""
+    input:
+        HTTP.remote("http://hgdownload.cse.ucsc.edu/goldenpath/hg19/database/cytoBand.txt.gz")
```
Comment (dustinbleile) [inline Snakefile]: couldn't this also replace the 'hg19' with:
"http://hgdownload.cse.ucsc.edu/goldenpath/{hgver}/database/cytoBand.txt.gz".format(hgver=config["genome_name"])

File: Snakefile [hunk 2]
```diff
@@ -434,7 +448,7 @@ rule ploidetect_copynumber:
     conda:
         "conda_configs/r.yaml"
     container:
-        "docker://lculibrk/ploidetect"
+        "docker://lculibrk/ploidetect:devel"
```
Comment (dustinbleile) [inline Snakefile]: Does this actually still need the different docker container? (the :devel?)

Comment (lculibrk) [reply Snakefile]: Didn't mean to push that, whoops

File: Snakefile (hunk 1 above)
Comment (lculibrk) [reply Snakefile]: I even included the relevant functionality in the output, and not the input. Will fix.

File: Snakefile (hunk 2 above)
Comment (lculibrk) [reply Snakefile]: But it reminds me that I need to rebuild & retest the base image after some tangles I got myself into on the Ploidetect repo

Comment (lculibrk) [reply Snakefile]: Reverted in 12bee71

File: Snakefile (hunk 1 above)
Comment (lculibrk) [reply Snakefile]: Resolved in 12bee71

**Review by dustinbleile — APPROVED** [review]

**Pull request merged**